openfl-tutorials/interactive_api/Flax_CNN_CIFAR/README.md

```diff
@@ -0,0 +1,87 @@
+# Federated FLAX CIFAR-10 CNN Tutorial
+
+### 1. About dataset
+
+The CIFAR-10 dataset consists of 60000 32x32 colour images in 10 classes, with 6000 images per class. There are 50000 training images and 10000 test images.
+
+Define the below param in envoy.yaml config to shard the dataset across participants/envoy.
+- rank_worldsize
+
+### 2. About model
+
+A simple multi-layer CNN is used with XLA compiled and Auto-grad based parameter updates.
+Definition provided in the notebook.
+
+### 3. How to run this tutorial (without TLS and locally as a simulation):
+
+0. Pre-requisites:
+
+    Activate virtual environment (Python - 3.8.10) and install below packages.
+
+```
+pip install openfl==1.3 protobuf==3.19.4 --no-cache-dir
```

Comment:
Can this be merged into `pip install -r requirements.txt`? Additionally, my sense is that requirements should be unique to an example (and therefore should be in the root directory of the example, not within envoy/director dirs).